src/pudl/etl/__init__.py

```diff
@@ -108,6 +112,9 @@
     ),
 )
 
+asset_check_modules = [pudl.transform.eia860, pudl.transform.eia923]
```

Comment:
In the future we probably want to do something like "define a list of all modules we would want to get asset checks *or* assets from, and then initialize `default_asset_checks` and `default_assets` from that single list." But. That is out of scope of this PR.